### PR TITLE
Updated to work on Openshift

### PR DESCRIPTION
**Pull request by paulrobinson**
This is now ready for review. 

There are a couple of untested tidy-ups as Openshift is currently down for maintenance:

OpenShift will be performing an update on Wednesday Mar 21st, some services may be unavailable.

### CODE REVIEW

File: wsat-simple/src/main/java/org/jboss/as/quickstarts/wsat/simple/servlet/WSATSimpleServletClient.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        BindingProvider bindingProvider = (BindingProvider) client;
+        List<Handler> handlers = new ArrayList<Handler>(1);
+        handlers.add(new JaxWSHeaderContextProcessor());
+        bindingProvider.getBinding().setHandlerChain(handlers);
+        //Lookup the DNS name of the server from the environment and set the endpoint address on the client.
+        String openshift = System.getenv("OPENSHIFT_APP_DNS");
+        if (openshift != null) {
+            bindingProvider.getRequestContext().put(BindingProvider.ENDPOINT_ADDRESS_PROPERTY, "http://"+openshift+"/wsat-simple/RestaurantServiceAT");
+        }
+
```
Comment (paulrobinson) [inline wsat-simple/src/main/java/org/jboss/as/quickstarts/wsat/simple/servlet/WSATSimpleServletClient.java:61]: I'm not happy with this part of the code. There should be a way of configuring the endpoint address through configuration. Maybe this is OK to leave in for the baseline?

Comment (pmuir) [inline wsat-simple/src/main/java/org/jboss/as/quickstarts/wsat/simple/servlet/WSATSimpleServletClient.java:61]: So basically what you are doing is relying on a system property that openshift always sets up?

Do we use this servlet now on localhost as well?

Comment (paulrobinson) [inline wsat-simple/src/main/java/org/jboss/as/quickstarts/wsat/simple/servlet/WSATSimpleServletClient.java:61]: Correct. 

You can use this Servlet locally too, but the readme only tells the user how to run the Arq test.

**Comment by pmuir** [comment]
I updated the README to align with Mike's and merged. Great work!

**Pull request closed without merging**